doc/nrf/known_issues.rst

```diff
@@ -979,6 +979,79 @@ KRKNWK-8133: CSMA-CA issues
 =====================
 
+.. rst-class:: v1-6-0
+
+DRGN-15817: Possible assert when used together with 802.15.4 protocols when running on nRF53 series.
+  The SoftDevice Controller may in some cases assert on nRF53 series products when used together with 802.15.4 protocols.
+
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+
+DRGN-15338: Extended scanner may generate reports containing truncated data from chained advertising PDUs
+  The scanner reports partial data from advertising PDUs when there is not enough storage space for the full report.
+
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+
+DRGN-15586: "HCI LE Set Scan Paramters" accepts a scan window greater than the scan interval
+  This can result in undefined behaviour. It should fail with the result "Invalid HCI Command Parameters (0x12)".
+
+  **Workaround:** The application should make sure the scan window is set to less than or equal to the scan interval.
+
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
+
+DRGN-15547: Assertion when updating PHY and the event length is configured too low
+  The SoftDevice Controller may assert if:
+
+  * :c:union:`sdc_cfg_t` with :c:member:`event_length` is set to less than 2500 us and the PHY is updated from 2M to 1M, or from either 1M or 2M to Coded PHY.
+  * :c:union:`sdc_cfg_t` with :c:member:`event_length` is set to less than 7500 us and a PHY update to Coded PHY is performed.
+
+  | The default value of :option:`CONFIG_SDC_MAX_CONN_EVENT_LEN_DEFAULT` is 7500 us.
+  | The minimum event length supported by :option:`CONFIG_SDC_MAX_CONN_EVENT_LEN_DEFAULT` is 2500 us.
+
+  **Workaround:**
+    * Set :c:union:`sdc_cfg_t` with :c:member:`event_length` to at least 2500 us if the application is using 1M PHY.
+    * Set :c:union:`sdc_cfg_t` with :c:member:`event_length` to at least 7500 us if the application is using Coded PHY.
+
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+
+DRGN-13594: The channel map provided by the LE Host Set Channel Classification HCI command is not always applied on the secondary advertising channels
+  In this case, the extended advertiser can send secondary advertising packets on channels which are disabled by the Host.
+
+.. rst-class:: v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+
+DRGN-15469: Slave connections can disconnect prematurely if there are scheduling conflicts with other roles
+  This is more likely to occur during long-running events such as flash operations.
+
+.. rst-class:: v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
+
+DRGN-15369: Setting radio output power using the vendor-specific HCI command Zephyr Write TX Power Level returns "Unsupported Feature or Parameter value (0x11)" if the chosen power level is not supported by the used hardware platform
```

Comment:
This entry does not build correctly. I suggest moving some of the information from this sentence to a new sentence so the entry has a proper second line and the build works.